packages/grafana-ui/src/components/Forms/Input/Input.tsx

```diff
@@ -11,9 +11,9 @@ export interface Props extends Omit<HTMLProps<HTMLInputElement>, 'prefix' | 'siz
   /** Show an invalid state around the input */
   invalid?: boolean;
   /** Show an icon as a prefix in the input */
-  prefix?: JSX.Element | string | null;
+  prefix?: ReactNode;
```

Comment:
@dprokop is it ok to change this? Otherwise it clashed with types used in the jaeger components but I assume this make more sense as this means anything render-able by react can be used.

Reply:
Yes, it makes sense, thank you